shinydashboard/lantern/modules/validationsmodule.R

```diff
@@ -0,0 +1,167 @@
+library(DT)
+library(purrr)
+
+validationsmodule_UI <- function(id) {
+
+  ns <- NS(id)
+
+  tagList(
+    fluidRow(
+      column(width = 3,
+             h3("Validation Details"),
+             DT::dataTableOutput(ns("validation_details_table"))
+            ),
+      column(width = 9,
+          fluidRow(
+            column(width = 12,
+                h3("Validation Results Count"),
+                uiOutput(ns("validation_results_plot"))
+            )
+          ), fluidRow(
+            column(width = 12,
+              h3("Validation Failure Details"),
+              DT::dataTableOutput(ns("validation_failure_table"))
+            )
+          )
+      )
+    )
+  )
+}
+
+validationsmodule <- function(
+  input,
+  output,
+  session,
+  sel_fhir_version,
+  sel_vendor,
+  sel_validation_group
+) {
+  ns <- session$ns
+
+  output$endpoint_count <- renderText({
```

Comment:
This doesn't look like it's being used?

Reply:
Yepp whoops that was a typo when I copied something else over, good catch!